src/main/java/com/pnu/ordermanagementapp/model/User.java

```diff
@@ -0,0 +1,64 @@
+package com.pnu.ordermanagementapp.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.springframework.security.core.GrantedAuthority;
+import org.springframework.security.core.authority.SimpleGrantedAuthority;
+import org.springframework.security.core.userdetails.UserDetails;
+
+import javax.persistence.*;
+import java.util.Collection;
+import java.util.Collections;
+
+@Data
+@Builder(toBuilder = true)
```

Comment:
Use only the needed annotations. It is bad for code coverage. It is regarding @Data annotation.

Reply:
Not a big difference here.
Regarding code coverage:
1. We don't have tests ;)
2. We have to try to cover as much business logic as possible (>70% is good coverage, but it's better to have coverage ~90%), data models isn't the kind of stuff we want to test, so having implicit setter (that doesn't clutter up our code), that we never use won't make a big differance

Reply:
>80% is good)